pkg/ddevapp/templates.go

```diff
@@ -5,7 +5,7 @@ package ddevapp
 const DDevComposeTemplate = `version: '2'
```

Comment:
Version 3.x does not support extends yet, so we'll want to stick to 2.x for now.